features/support/env.rb

```diff
@@ -1,5 +1,18 @@
 # Configure app environment
 
+# Set which test fixture should be used
+if ENV["APP_FIXTURE_DIR"] == nil
+  ENV["APP_FIXTURE_DIR"] = "features/fixtures/app"
+end
+
+if ENV["NDK_FIXTURE_DIR"] == nil
+  ENV["NDK_FIXTURE_DIR"] = "features/fixtures/ndkapp"
+end
+
+if ENV["RN_FIXTURE_DIR"] == nil
+  ENV["RN_FIXTURE_DIR"] = "features/fixtures/rnapp/android"
+end
```

Comment:
This could be shorter, if it's not too unclear:

```suggestion
ENV["APP_FIXTURE_DIR"] ||= "features/fixtures/app"
ENV["NDK_FIXTURE_DIR"] ||= "features/fixtures/ndkapp"
ENV["RN_FIXTURE_DIR"] ||= "features/fixtures/rnapp/android"
```

`||=` will only do the assignment if the current value is falsy

Reply:
Nice :) I did not know this